internal/storage/gcs/bucket.go

```diff
@@ -20,6 +20,36 @@ import (
 	"golang.org/x/net/context"
 )
 
+// ObjectWriter interface represents asynchronous object
+// writers, which don't write the entire GCS object from
+// content in a single-shot. These objects are created once,
+// are written to multiple times and are eventually closed
```

Comment:
The comment doesn't look right to me. You are mentioning that objects are created once and written to multiple times, gives the impression that object is written/updated in every write. instead mention it clearly that this interface is used to upload object using resumable api.

Reply:
Corrected. PTAL.